EDMarketConnector.py

```diff
@@ -1582,7 +1582,7 @@ def crewroletext(role: str) -> str:
                 if not should_return:
                     self.w.after(int(SERVER_RETRY * 1000), self.capi_request_data)
 
-            if entry['event'] in ('CarrierBuy', 'CarrierStats'):
+            if entry['event'] in ('CarrierBuy', 'CarrierStats') and config.get_int('capi_fleetcarrier') == 1:
```

Comment:
It'll end up *stored* as in int on Windows, but you should be using `get_bool()` and sending it a `bool` value in `config.set()`.